packages/angular_devkit/schematics/tasks/run-schematic/init-task.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { TaskConfiguration, TaskConfigurationGenerator } from '../../src';
+import { RunSchematicName, RunSchematicTaskOptions } from './options';
+
+
+export class RunSchematicTask implements TaskConfigurationGenerator<RunSchematicTaskOptions> {
+  constructor(
+    protected _collection: string,
```

Comment:
should this support the combined string format for the collection and schematic as well?

Reply:
The goal of this task is to be created/ran from code. The `collection:schematic` syntax is just convention from command line.